--- origsrc/lxterminal-0.1.4/src/lxterminal.c	2008-12-26 00:35:21.000000000 -0600
+++ src/lxterminal-0.1.4/src/lxterminal.c	2009-02-15 16:41:30.125000000 -0600
@@ -56,6 +56,7 @@ const GdkColor linux_color[16] =
 };
 
 LXTerminal *lxterminal_init(LXTermWindow *lxtermwin, gint argc, gchar **argv, Setting *setting);
+void terminal_childexit(VteTerminal *vte, Term *term);
 
 static gchar helpmsg[] = {
 	"Usage:\n"
